pkg/http/http.go

```diff
@@ -284,6 +284,18 @@ func (t *TransportWrapper) logResponse(resp *http.Response) {
 	log.Entry().Debug("--------------------------------")
 }
 
+func transformHeaders(header http.Header) http.Header {
+	var h http.Header = map[string][]string{}
+	for name, value := range header {
+		if name == "Authorization" {
+			value = []string{"<set>"}
```

Comment:
rather use secret registration: `log.RegisterSecret(theSecret)`?

Reply:
So we have to

* concatenate `<user>` + `:` + `<passwd>`
* calculate the base64 hash
* register that hash as as secret.

Seems to be a lot of overhead to me compared to just ensure that the value is overwritten here.

Reply:
The secret is registered above as plan text? Does it not need base64 encoding in case of basic auth?

Reply:
>The secret is registered above as plan text?

No. We have an auth header: `Authorization: Basic SSBsaWtlIFNBUAo=` and we register the base64 encoded value (here `SSBsaWtlIFNBUAo=`).

Reply:
Because the second value `v` is already base64 encoded, got it.